django_ltree/fields.py

```diff
@@ -7,7 +7,7 @@
 from collections.abc import Iterable
 
 path_label_validator = RegexValidator(
-    r"^(?P<root>[a-zA-Z][a-zA-Z0-9_]*|\d+)(?:\.[a-zA-Z0-9_]+)*$",
+    r"^(?P<root>[a-zA-Z0-9_]+)(?:\.[a-zA-Z0-9_]+)*$",
```

Comment:
I know you mentioned ignoring the hyphen, but I think it’s worth matching what is currently supported. This also aligns with allowing strings that are slugified using Django's utilities.

```suggestion
    r"^(?P<root>[a-zA-Z0-9_-]+)(?:\.[a-zA-Z0-9_-]+)*$",
```